### DISCUSSION TITLE
Option for "Maximize" to ignore selected gap between windows

### DISCUSSION THREAD

**Discussion by ridjis** [rxhanson/Rectangle#1046] [Ideas]
First of all, thanks for this app!

I'm curious if there's a possibility to add option in Preferences to ignore specified gap when "Maximize" option is selected (via menu or shortcut).

I like gap between windows feature when doing split windows, but reverted it to 0 because I often use "Maximize" shortcut and that padding around maximized window bothers me 😃

**Comment by rxhanson** [comment]
Is this what you're looking for?

https://github.com/rxhanson/Rectangle/blob/master/TerminalCommands.md#disabling-gaps-when-maximizing

**Reply by ridjis** [reply]
Wow, wasn't aware of those additional options. Thanks, that's it!
